commands/whoami.go

```diff
@@ -3,13 +3,49 @@ package commands
 import (
 	"fmt"
 	tgbotapi "github.com/go-telegram-bot-api/telegram-bot-api/v5"
+	"github.com/sxyazi/bendan/utils"
+	"io"
+	"log"
+	"net/http"
+	"regexp"
 )
 
+// This DC query method is based on checking the user profile photos.
+// It may not work if the user doesn't have a username or profile photos or their photos is hidden.
+func dcQuery(username string) string {
+	req, err := http.NewRequest("GET", fmt.Sprintf("https://t.me/%s", username), nil)
+	if err != nil {
+		log.Printf("DC Query: Error occurred while creating HTTP request: %v\n", err)
+		return ""
+	}
+	resp, err := utils.Client.Do(req)
+	if err != nil {
+		log.Printf("DC Query: Error occurred while sending HTTP request: %v\n", err)
+		return ""
+	}
+	defer resp.Body.Close()
+	body, err := io.ReadAll(resp.Body)
+	if err != nil {
+		log.Printf("DC Query: Error occurred while reading response body: %v\n", err)
+		return ""
+	}
```

Comment:
Maybe it would be better to use [getUserProfilePhotos](https://core.telegram.org/bots/api#getuserprofilephotos) here instead of requests directly?

Reply:
Let me try it.